src/modules/tracker.js

```diff
@@ -579,32 +598,39 @@ class Tracker {
    * constructorio.tracker.trackSearchResultsLoadedV2(
    *     'T-Shirt',
    *     {
-   *         result_count: 167,
-   *         items: [{item_id: 'KMH876'}, {item_id: 'KMH140'}, {item_id: 'KMH437'}],
-   *         sort_order: 'ascending'
-   *         sort_by: 'price',
-   *         result_page: 3,
-   *         result_count: 20
+   *         resultCount: 167,
+   *         items: [{itemId: 'KMH876'}, {itemId: 'KMH140'}, {itemId: 'KMH437'}],
+   *         sortOrder: 'ascending'
+   *         sortBy: 'price',
+   *         resultPage: 3,
+   *         resultCount: 20
    *     },
    * );
    */
-  trackSearchResultsLoadedV2(search_term, parameters, networkParameters = {}) {
+  trackSearchResultsLoadedV2(searchTerm, parameters, networkParameters = {}) {
     // Ensure term is provided (required)
-    if (search_term && typeof search_term === 'string') {
+    if (searchTerm && typeof searchTerm === 'string') {
       // Ensure parameters are provided (required)
       if (parameters && typeof parameters === 'object' && !Array.isArray(parameters)) {
         const baseUrl = `${this.behavioralV2Url}search_result_load?`;
         const {
           num_results,
-          result_count = num_results,
+          numResults = num_results,
```

Comment:
Similarly here, do we need the numResults?

Reply:
Same as above